docs/package.json

```diff
@@ -14,14 +14,14 @@
     "serve": "serve ./export -l 3010",
     "create-playground": "cpy --cwd=scripts playground.template.tsx ../../pages/playground --rename=index.tsx",
     "typescript": "tsc -p tsconfig.json",
-    "typescript:transpile": "cross-env BABEL_ENV=development babel-node --extensions \".tsx,.ts,.js\" scripts/formattedTSDemos",
+    "typescript:transpile": "cross-env BABEL_ENV=development node scripts/formattedTSDemos",
```

Comment:
The `--disable-cache` argument failed to be passed into the script after the bump:
https://app.circleci.com/pipelines/github/mui/mui-x/59074/workflows/cafb4bed-14c0-499b-a4b0-b20d6f76cb68/jobs/337511
I didn't look too deep into the root cause, because just changing to `node` as the script executor fixed this problem. 🙈 🤷 
As we do not want to over-invest in relying on Babel, I didn't see the point in investigating, why it is actually happening.